src/pyuvdata/uvbeam/uvbeam.py

```diff
@@ -668,6 +803,7 @@ def _set_power(self):
         self._Ncomponents_vec.required = False
         self._Nfeeds.required = False
         self._feed_array.required = False
+        self._feed_angle.required = False
```

Comment:
I think we talked about always having feed_array being required, even for power beams.

Reply:
Sorry -- wasn't clear if we wanted to also fold that into this PR, changed in 0c9851936cf3aae64c073b4df8b527903988922f.